src/ide.js

```diff
@@ -106,9 +101,9 @@ function formatIdeResult(result, options, index, length) {
       // strip newlines at the end
       if (endsOnNewline) {
         trimmed = lines.reverse().reduce((trimmed, line, i) => {
```

Comment:
Unrelated question: wouldn't this be better as `lines.reduceRight` over `line.reverse().reduce`?